### PR TITLE
Projects redesign - Map Feature Explorer

### PR DESCRIPTION
**Pull request by mohitb35**
Changes:

1. Sets up MapFeatureExplorer, refactoring ExploreProject empty component (storybook), including renaming and refactoring subcomponents.
2. Adds show / hide functionality for project markers on the map
3. Minor style adjustments
4. Comments out unready features within MapSettings menu

### CODE REVIEW

**Comment by sunilsabatp** [comment]
image
Could you address the above error ?
-> To reproduce the error, click on the explore feature after the page has initially loaded.

**Review by sunilsabatp — CHANGES_REQUESTED** [review]

File: src/features/projectsV2/ProjectsMap/MapFeatureExplorer/MapFeatureExplorer.module.scss [hunk 1]
```diff
@@ -7,6 +7,7 @@
   margin-left: 3px;
   padding-top: 14px;
   padding-bottom: 14px;
+  box-shadow: 0px 6px 10px rgba(0, 0, 0, 0.12);
```
[the remark this answers is not in the document]
Comment (mohitb35) [reply src/features/projectsV2/ProjectsMap/MapFeatureExplorer/MapFeatureExplorer.module.scss:10]: The shadow applied to the button does not get applied here, as both CustomButton and MapSettings get their styles from separate places.

[the remark this answers is not in the document]
Comment (mohitb35) [reply src/features/projectsV2/ProjectsMap/MapFeatureExplorer/MapFeatureExplorer.module.scss:10]: The shadow applied to the button does not get applied here, as both CustomButton and MapSettings get their styles from separate places.

**Comment by mohitb35** [comment]
> 948034b

Resolved in [948034b](https://github.com/Plant-for-the-Planet-org/planet-webapp/pull/2164/commits/948034bb52b10057bd1d0e80716da5cadf35ba66)

**Comment by mohitb35** [comment]
> image
> Could you address the above warning ? -> To reproduce the error, click on the explore feature after the page has initially loaded.

Resolved in [948034b](https://github.com/Plant-for-the-Planet-org/planet-webapp/pull/2164/commits/948034bb52b10057bd1d0e80716da5cadf35ba66)

**Review by sunilsabatp — APPROVED** [review]

**Pull request merged**